index.bs

```diff
@@ -1903,6 +1916,16 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
     [=authenticators=] can be <a href="https://en.wikipedia.org/w/index.php?title=Hot_plug">hot-plugged</a> into (e.g., via USB)
     or discovered (e.g., via NFC or Bluetooth) by the [=client=] by various mechanisms, or permanently built into the [=client=].
 
+1. If <code>|options|.{{CredentialRequestOptions/mediation}}</code> is present with the value
+     {{CredentialMediationRequirement/conditional}}:
+
+    1. If |conditionalCreateLifetimeTimer| is expired or |conditionalCreateOrigin| is not |callerOrigin|, throw a "{{NotAllowedError}}" {{DOMException}}.
+
+    Note: |conditionalCreateLifetimeTimer| and |conditionalCreateOrigin| will be set by the user agent after it believes an authentication ceremony has
+    been completed and the user consents to this type of credential creation.
```

Comment:
Is this specified anywhere? I can't see any other references to `conditionalCreateLifetimeTimer` or `conditionalCreateOrigin` either here or in https://github.com/w3c/webappsec-credential-management/pull/224 .

Reply:
It was defined in the extension, which got removed. I reworded this section to not use these variables and instead specify that the user agent must believe an authentication ceremony recently took place and that the user consents to the credential creation.